packages/design-system/src/components/typography/headline/index.js

```diff
@@ -46,7 +46,7 @@ export const Headline = styled.h1`
 `;
 
 Headline.propTypes = {
-  as: PropTypes.oneOf(['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'a']),
+  as: PropTypes.oneOf(['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'a', 'label']),
```

Comment:
💭  Doesn't the `label` already exist as an option of `<Text>`?

https://github.com/GoogleForCreators/web-stories-wp/blob/d16f28fb45d808bf5abef2a5ae2c69feb5a63f66/packages/design-system/src/components/typography/text/index.js#L88-L94

Does the headline need this option?

Reply:
😢 design has the line spacing matching the `Headline` instead of the `Text` so i added a label option there.  I should check with Amy though, maybe it was an accident or not important.